MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -201,14 +201,17 @@ private async Task FetchDataFromTags(Audio audio, Model.MediaInfo.MediaInfo medi
 
                 foreach (var albumArtist in albumArtists)
                 {
-                    if (!string.IsNullOrWhiteSpace(albumArtist))
```

Comment:
`string.IsNullOrWhiteSpace()` already handles the case where the string will be null or empty after trimming. No need to do extra check here.